sdk/Pulumi.Automation/Commands/CommandResult.cs

```diff
@@ -4,7 +4,7 @@
 
 namespace Pulumi.Automation.Commands
 {
-    internal class CommandResult
+    public class CommandResult
```

Comment:
Others may roll their eyes, but we may want to consider making this sealed. I don't realistically see anyone wanting to inherit from this and sealing it gives us options to make changes without breaking people. If we ever want to allow people to subclass, we can consider unsealing.

```suggestion
    public sealed class CommandResult
```